v3/go.sum

```diff
@@ -1,6 +1,6 @@
 github.com/Duet3D/DSF-APIs v0.0.0-20200909082542-eff4037e650f h1:1/l3Q6p94fNCdWj3F9buHzO7Hgl4lxGqjTxAHPBdV9s=
 github.com/Duet3D/DSF-APIs v0.0.0-20200909135208-e1377604004b h1:4mIh5gaA1syBi26QQKMIavCIhkCQTBCnMTLOyQHdXK0=
 github.com/Duet3D/DSF-APIs v0.0.0-20200910090227-45f68893a13a h1:Swt2fD7xRvOjvkwn4j67z65kTtM2P54PkChD5JWlEgE=
-github.com/Duet3D/DSF-APIs/godsfapi v1.2.4 h1:igiN18KMXL9l6yGJ4ms1W8NX9hUvvS4m1C+Yh0uvEKI=
+github.com/Duet3D/dsf-go v1.2.4 h1:igiN18KMXL9l6yGJ4ms1W8NX9hUvvS4m1C+Yh0uvEKI=
```

Comment:
I don't think this line should have been in here in the first place, please remove it entirely.

Reply:
what about the dsf-apis lines? what is this file for?

Reply:
Best would be to let go cleanup itself. Run
```
go mod tidy
```
in the directories with `go.mod` files. That will remove everything that does not belong there.

Reply:
deleted all duet3d lines from sums file. right?